rules/book-developers-for-a-project/rule.md

```diff
@@ -88,9 +87,9 @@ Sometimes you may want to reserve a consultant for an appointment but are not ab
 To create a Tentative Booking:
 
 1. Follow the same steps above to create the Appointment
-2. Instead of choosing a client account for 'Set Regarding', choose the company (E.g. "ssw.pencilledin").
+2. Instead of choosing a client account for 'Set Regarding', choose the client you have created for the purpose of tentative bookings (E.g.**ssw.pencilledin**)
```

Comment:
```suggestion
2. Instead of choosing a client account for 'Set Regarding', create and use a pre-defined account for the purpose of tentative bookings (E.g.**SSW.PencilledIn**)
```